client/src/index.jsx

```diff
@@ -17,5 +17,5 @@ render((
     <Route path="/login" component={Login} />
     <Route path="/logout" component={Logout} onEnter={requireAuth} />
   </Router>
-  ), document.getElementById('app')
+  ), document.getElementById('app'),
```

Comment:
Okay, I updated the rule to specify `never` for `functions` and `always-multiline` for all other types.